tests/ha/single_pod_restart/test_data_server_pod_restart.py

```diff
@@ -96,23 +97,57 @@ def setup_method(self):
         self.restore_node = False
         self.restore_ip = False
         self.s3_clean = dict()
+        self.restore_pod = self.restore_method = self.deployment_name = self.set_name = None
+        self.deployment_backup = None
+        self.num_replica = 1
         self.s3acc_name = f"ha_s3acc_{int(perf_counter_ns())}"
         self.s3acc_email = f"{self.s3acc_name}@seagate.com"
         LOGGER.info("Precondition: Verify cluster is up and running and all pods are online.")
         resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Precondition: Verified cluster is up and running and all pods are online.")
+        LOGGER.info("Get %s and %s pods to be deleted", const.POD_NAME_PREFIX,
+                    const.SERVER_POD_NAME_PREFIX)
+        self.combine_delete_pod = list()
+        self.combine_set_name = list()
```

Comment:
Instead of two lists, can you create a dict which will have:
{'delete_pod_name': ['set_name, num_replicas]}
This way we will have all theinfo of the pods to be deleted at one place.